src/openassetio-core-c/include/openassetio/c/StringView.h

```diff
@@ -14,10 +14,25 @@ extern "C" {
  * @{
  */
 
-// Symbol namespacing.
+/**
+ * @defgroup oa_StringView oa_StringView
+ *
+ * C API for views on character buffers.
+ *
+ * @{
+ */
+
+/**
+ * @defgroup StringView Aliases
```

Comment:
[fix] `StringView` -> `oa_StringView_aliases`

Reply:
Fixed in 2b654545